rg_instructor_analytics/management/commands/update_rg_analytics_precollection_data.py

```diff
@@ -0,0 +1,24 @@
+"""
+Manage.py command for the update precollected data.
+"""
+from django.core.management.base import BaseCommand
+
+from rg_instructor_analytics.tasks import run_common_static_collection
+
+
+class Command(BaseCommand):
+    """
+    Command for initial collect data for the rg_instructor_analytics.
+    """
+
+    help = """
+    Send task to celery for update internal data about platform.
+
+    Attention - collection of the initial data take several hours.
```

Comment:
s/Attention - collection of the initial data take several hours./Attention - the first run of the collecting data may take several hours./